setup.py

```diff
@@ -98,63 +98,9 @@ def zip_safe(self):
     extra['include_package_data'] = True
 
 
-# include tests for python3 setup.py test (needed when creating
-# source distributions on python2 too so that they work on python3)
-packages = [
-    'genshi', 'genshi.filters', 'genshi.template',
-    'genshi.tests', 'genshi.filters.tests',
-    'genshi.template.tests',
-    'genshi.template.tests.templates',
-]
-
-
 setup(
-    name = 'Genshi',
-    version = '0.8',
-    description = 'A toolkit for generation of output for the web',
-    long_description = \
-"""Genshi is a Python library that provides an integrated set of
-components for parsing, generating, and processing HTML, XML or
-other textual content for output generation on the web. The major
-feature is a template language, which is heavily inspired by Kid.""",
-    author = 'Edgewall Software',
-    author_email = 'info@edgewall.org',
-    license = 'BSD',
-    url = 'https://github.com/edgewall/genshi',
-
-    classifiers = [
-        'Development Status :: 4 - Beta',
-        'Environment :: Web Environment',
-        'Intended Audience :: Developers',
-        'License :: OSI Approved :: BSD License',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 3',
-        'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
-        'Topic :: Software Development :: Libraries :: Python Modules',
-        'Topic :: Text Processing :: Markup :: HTML',
-        'Topic :: Text Processing :: Markup :: XML'
-    ],
-    keywords = ['python.templating.engines'],
-    packages = packages,
     test_suite = 'genshi.tests.suite',
```

Comment:
I don't recall if it's been discussed in this project yet, but the [test](https://setuptools.readthedocs.io/en/latest/userguide/commands.html#test) command is deprecated.

Reply:
That should really be its own issue, and actually this coincides very well with the other mention here about reorganizing into a tests/ directory.

Reply:
Agreed -- let's leave messing with the test suite out of things for now.